invenio/base/scripts/apache.py

```diff
@@ -17,194 +17,414 @@
 ## along with Invenio; if not, write to the Free Software Foundation, Inc.,
 ## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
+"""Apache configuration generator."""
+
 from __future__ import print_function
 
+import os
+import pwd
+import re
+import sys
+from distutils import dir_util
+from os import path
+
+import pkg_resources
+import shutil
+import subprocess
+from flask import current_app as app
```

Comment:
please use `current_app` for consistency

- `current_app` - proxy to application object in current app context
- `app` - usually an instance of `Flask`